Makefile

```diff
@@ -50,6 +50,8 @@ update:
 #    make verify
 verify:
 	@which go-bindata 2> /dev/null >&1 || { echo "go-bindata must be installed to verify generated code";  exit 1; }
+	@which golangci-lint 2> /dev/null >&1 || { echo "golangci-lint must be installed to lint code";  exit 1; }
```

Comment:
@runcom qq - is it bound to a particular release ?

Reply:
i suspect it is by looking https://github.com/openshift/release/pull/3799/files so maybe the echo can be extended to mention that ... my 0.02$ though

Reply:
closed that PR as it's not needed now that we're all above go1.10 (1.12 to be precise)

Reply:
How about adding a target to install it for you? e.g.

```
$(GOBIN)/golangci-lint: 
	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint

verify: $(GOBIN)/golangci-lint
```

Reply:
I've gona back and forth on the target (we used to have it in CRI-O as well but ended up being dropped cause the CI was taking care of it https://github.com/cri-o/cri-o/pull/2247/files#diff-b67911656ef5d18c4ae36cb6741b7965L308)

The real pain with go1.12 and onwards would be modules also (and calculating $GOBIN). I'm not yet familiar with that. I'm super fine following up to this and add targets to install tools also 👍

Reply:
👍 I guess I'm used to having a golang version check to handle compatibility. there were issues with GOBIN? never come across that myself.

Reply:
> I guess I'm used to having a golang version check to handle compatibility.

oh cool, that might be something useful to have here in MCO as well :) 

> there were issues with GOBIN? never come across that myself.

afaict with go modules,  go build creates binaries in PWD so that must be conditionalized as well (hence your suggestion above to version check is amazing)